mods/ts/installer/downloads.yaml

```diff
@@ -1,4 +1,4 @@
-basefiles: Base Freeware Content
+tibsun: C&C Tiberian Sun
```

Comment:
Please keep the displayed names as they are.  The naming (a) makes it clear that the original games were released as freeware, and (b) keeps consistency between the mods.